setup.cfg

```diff
@@ -18,6 +18,8 @@ package_dir=
     =python
 packages=
     astro_metadata_translator
+    astro_metadata_translator.translators
```

Comment:
This is fine, but I'm interested for my own sake why this was needed. With my work I'm always doing `setuptools.find_packages` in a `setup.py` file so it must automatically be adding these sub packages for me. Automatic package detection doesn't work here I guess?

Reply:
I do have `options.packages.find` set later but it really was not installing these files at all until I explicitly mentioned them here. Maybe I'm doing it wrong?  Sometimes the migration from `setup.py` to `setup.cfg` is non-obvious but I am trying to follow the modern best practice here and define everything in the cfg file.

Reply:
Thank you for the prod. It made me look again and I am now using `find_packages` using the completely non-obvious syntax found in `setup.cfg`: see 4c5b6163cbac00abc7a7f84930888cefbc4ea95a